dapr/proto/scheduler/v1/scheduler_callback.proto

```diff
@@ -1,33 +0,0 @@
-syntax = "proto3";
```

Comment:
This is not needed anymore with the streaming solution being implemented by @cicoyle.